### ISSUE TITLE
Consider using .pivot_table instead of groupby/aggs

### ISSUE THREAD

**Issue by justalfred** [Datapolitan-Training/data-analysis-python#111]
https://pandas.pydata.org/pandas-docs/stable/user_guide/reshaping.html#pivoting-with-single-aggregations

Pros:

- most people know Excel, so this is a closer analogy than .groupby()...
- the syntax is tighter

Cons:

- I use it less, so I don't know the edge cases to watch out for.

Any other considerations?

**Comment by datapolitan** [comment]
I think this is good to consider. The syntax looks a little more complicated, but it might be easier for them to grasp than I think based on [this Stack Overflow question](https://stackoverflow.com/questions/34702815/pandas-group-by-and-pivot-table-difference). Happy to test and see.

**Comment by justalfred** [comment]
Yeah, how _would_ we test it? It would be a pretty significant rewrite of the materials, so I don't want us to dump that much work into something without some kind of smaller test.

hmm.... Maybe if we have someone ahead of the curve in the next class see if they can solve the same problems using it? If they find it more intuitive, then maybe others will, too? Any suggestions?
